scripts/create-darwin-volume.sh

```diff
@@ -631,7 +646,7 @@ EOF
         # technically /etc/synthetic.d/nix is supported in Big Sur+
         # but handling both takes even more code...
         _sudo "to add Nix to /etc/synthetic.conf" \
-            /usr/bin/ex /etc/synthetic.conf <<EOF
+            /usr/bin/ex --noplugin /etc/synthetic.conf <<EOF
```

Comment:
May be a bit more verbose, but would it be a good idea to use the wrapper here as well? So that these instances don't fall out of sync?

Reply:
I agree with the suggestion, and I'll draft something locally, but I'm feeling a little time-pressured to want to risk tacking it on here.

It's been over 5 weeks since the last release and I'm getting anxious about this fix getting in and released. The CI cycle time on these is about 1.5h at the moment, assuming the build doesn't flake, and I'll have limited time the next few days as we get our ducks in a row for a family wedding. 😬